rllib/env/base_env.py

```diff
@@ -496,7 +496,6 @@ def poll(self) -> Tuple[MultiAgentDict, MultiAgentDict, MultiAgentDict,
                     del self.last_infos[ag]
 
         self.last_dones["__all__"] = False
-        self.last_infos = {}
```

Comment:
what's the reason to remove this?

Reply:
@richardliaw,

To be consistent?  If you look at the whole method basically two things happen here wrt the last_* members. If the episode is done they are all dropped and replaced with empty dicts.  If the episode is not done then it releases only the values for specific agents in the observation.

Except for some reason right before the method returns it dropped the last_info dict and only the last_info the others (last_{obs, done, reward}) are left alone.

I figured this meant it was there on mistake.

Reply:
No that's a good explanation! thanks for the thorough reply :)